resources/views/partials/navbar/menu-item-dropdown-user-menu.blade.php

```diff
@@ -63,7 +63,7 @@ class="img-circle elevation-2"
         {{-- User menu footer --}}
         <li class="user-footer">
             @if($profile_url)
-                <a href="{{ $profile_url }}" class="btn btn-default btn-flat">
+                <a href="{{ $profile_url }}" class="nav-link btn btn-default btn-flat" style="display: inline-block">
```

Comment:
Nice catch! However, instead of `style="display: inline-block"`, please use the Bootstrap class `d-inline-block`, like this:

```blade
<a href="{{ $profile_url }}" class="nav-link btn btn-default btn-flat d-inline-block">
```

Thanks!

Reply:
Great!